.github/workflows/publish-1.1-docs.yml

```diff
@@ -0,0 +1,69 @@
+name: Publish 1.1 docs
+
+permissions:
+  contents: read
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish-docs:
+    # runs on main repo only
+    if: github.repository == 'apache/pekko-persistence-jdbc'
+    name: Publish
+    runs-on: ubuntu-20.04
+    env:
+      JAVA_OPTS: -Xms2G -Xmx2G -Xss2M -XX:ReservedCodeCacheSize=256M -Dfile.encoding=UTF-8
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+          fetch-tags: true
+
+      - name: Setup Java 8
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 8
+
+      - name: Build Documentation
+        run: |-
+          sbt "set ThisBuild / version := \"1.1.0-M1\"; docs/paradox; unidoc"
+
+      # Create directory structure upfront since rsync does not create intermediate directories otherwise
+      - name: Create directory structure
+        run: |-
+          mkdir -p target/nightly-docs/docs/pekko-persistence-jdbc/1.1.0-M1/
+          mkdir -p target/nightly-docs/docs/pekko-persistence-jdbc/1.1/
+          cp -r docs/target/paradox/site/main/ target/nightly-docs/docs/pekko-persistence-jdbc/1.1.0-M1/docs
+          cp -r docs/target/paradox/site/main/ target/nightly-docs/docs/pekko-persistence-jdbc/1.1/docs
+          rm -r docs/target/paradox/site/main/
+          cp -r target/scala-2.13/unidoc target/nightly-docs/docs/pekko-persistence-jdbc/1.1.0-M1/api
+          cp -r target/scala-2.13/unidoc target/nightly-docs/docs/pekko-persistence-jdbc/1.1/api
+          rm -r target/scala-2.13/unidoc
```

Comment:
Why we didn't publish japi docs?

Reply:
We only publish Java APIs for core and HTTP modules.

https://pekko.apache.org/modules.html

You can raise an issue but if you check Akka docs, it is basically the same for Akka (that specific Javadocs as opposed to Scaladocs are not published for all modules).

Reply:
It seems fair to me, I just asking.